ecommerce/frontend/src/components/ChooseListModalInner.js

```diff
@@ -0,0 +1,118 @@
+import './ChooseListModalInner.less'
+
+import React, { useEffect, useState } from 'react'
+import { Button, List, notification, Skeleton } from 'antd'
+import { sleep } from '../utils'
+import { HeartOutlined, HeartFilled } from '@ant-design/icons'
+
+export const ChooseListModalInner = ({ product }) => {
+    const [lists, setLists] = useState([])
+    const [loading, setLoading] = useState(true)
+    const [loadingList, setLoadingList] = useState(null)
+
+    const getLists = async (product, showLoading = true) => {
+        try {
+            setLoading(showLoading)
+            await sleep(1000)
+            // backend call to get lists of user
+
+            setLists([
+                {
+                    id: 1,
+                    name: 'Electronics',
+                    hasProduct: true,
+                },
+
+                {
+                    id: 2,
+                    name: 'Thanksgiving',
+                    hasProduct: false,
+                },
+                {
+                    id: 3,
+                    name: 'Blackfriday',
+                    hasProduct: true,
+                },
+            ])
+        } catch (error) {
+            notification.error({ description: 'Failed to get the lists' })
+        } finally {
+            setLoading(false)
+        }
+    }
+
+    useEffect(() => {
+        getLists(product, true)
+    }, [])
+
+    const removeProductFromList = async (product, list) => {
+        try {
+            setLoadingList(list)
+            await sleep(1000) // actual code here
+            notification.success({ description: 'Removed product from list' })
```

Comment:
I agree with you on not showing the success notification (since heart icon gets filled)